### DISCUSSION TITLE
Make the URL bar to allow direct typing.

### DISCUSSION THREAD

**Discussion by AndrewChalikias** [zen-browser/desktop#3774] [Ideas]
The URL bar could be clickable without opening the separate default Firefox bar (at least under the normal behavior setting) or minimize it a bit.
Like this:
![image](https://github.com/user-attachments/assets/824479e5-6c60-4b1d-bb20-f99a4bf7362f)
![image](https://github.com/user-attachments/assets/4b64d235-353c-4539-9faa-f2ed07ff74a6)

**Comment by chiangs** [comment]
you could just do `ctrl + L` and you will directly be in the address bar
